ui/desktop/app/templates/scopes/scope/projects/targets/target.hbs

```diff
@@ -24,7 +24,6 @@
   </page.actions>
 
   <page.body>
-    <div class='copy-url-container'>{{t 'actions.copy-to-clipboard'}}</div>
```

Comment:
Are we moving this copy button somewhere else?

Reply:
Yes sorry I should have added that context. The copy button will only exist on the Session Detail screen, so the Target details screen no longer needs this.